application/common/components/Emailer.php

```diff
@@ -24,7 +60,7 @@ class Emailer extends Component
      * @param string $subject The subject.
      * @param string $htmlBody The email body (as an HTML string).
      */
-    public function email(string $toAddress, string $subject, string $htmlBody)
+    protected function email(string $toAddress, string $subject, string $htmlBody)
```

Comment:
is there a reason we're only supporting html emails? the mailer service supports html and text and I assumed we'd render both versions